intranet/apps/eighth/tests/admin/test_admin_scheduling.py

```diff
@@ -173,6 +173,68 @@ def test_schedule_activity_view(self):
             0, EighthScheduledActivity.objects.filter(block=block, activity=activity, capacity=5, cancelled=True, both_blocks=True).count()
         )
 
+    def test_room_change_notification_view(self):
+        """Tests :func:`~intranet.apps.eighth.views.admin.scheduling.schedule_activity_view`."""
```

Comment:
I believe this comment needs to be changed? -- specifically, to state that it only tests for room change notification?